src/styles.scss

```diff
@@ -1398,6 +1398,168 @@ div.mat-menu-panel {
   min-height: min-content;
 }
 
+// "Formatted notebook" styles, which are here because the HTML is dynamically generated
+// see https://stackoverflow.com/questions/56265483/angular-how-to-style-dynamically-generated-html
+.notebook {
+  display: grid;
+  grid-template-columns: fit-content(7em) auto;
+  grid-template-rows: auto;
+  grid-auto-flow: row;
+  padding: 1em 1.5em 1em 1em;
+  border: solid 1px #cfd1d7;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.1);
+  border-radius: 4px;
+  line-height: 1.5;
+  font-size: 12px;
+
+  h1 {
+    font-size: 24px;
+  }
+  h2 {
+    font-size: 20px;
+  }
+  h3 {
+    font-size: 16px;
+  }
+  code,
+  pre {
+    font-size: 12px;
+  }
+  ul,
+  ol,
+  dl,
+  li,
+  table {
+    font-size: 100%;
+  }
+
+  table td,
+  table th {
+    padding: 1px 3px;
+  }
+
+  .count {
+    grid-column: 1;
+    margin: 0.3em;
+    padding: 1em 0.2em 0 0;
+    text-align: right;
+    color: #aaa;
+    line-height: 1.5;
+  }
+
+  .markdown,
+  .source,
+  .output {
+    grid-column: 2;
+    margin: 0.3em;
+    overflow-x: auto;
+    overflow-y: hidden;
+  }
+
+  .count,
+  .source,
+  .output {
+    font-family: Menlo, Monaco, Consolas, 'Courier New', monospace;
+    white-space: pre;
+  }
+
+  .markdown,
+  .output {
+    padding: 0;
+  }
+
+  .markdown p:first-child {
+    margin-top: 0;
+  }
+
+  .markdown p:last-child {
+    margin-bottom: 0;
+  }
+
+  .source {
+    padding: 1em;
+    background-color: #fcf8fa;
+    border: solid 1px #f2edf0;
+  }
+
+  .source pre,
+  .output pre {
+    padding: 0;
+    margin: 0;
+    border: none;
+    background-color: inherit;
+    color: inherit;
+  }
+
+  // These rules target the html generated by the Prism syntax highlighter.
+  // This particular style was derived from a css file from the Prism distribution.
+  .source {
+    .token.comment,
+    .token.prolog,
+    .token.doctype,
+    .token.cdata {
+      color: #8ac;
+    }
+
+    .token.punctuation {
+      opacity: 0.7;
+    }
+
+    .token.namespace {
+      opacity: 0.7;
+    }
+
+    .token.property,
+    .token.tag,
+    .token.boolean,
+    .token.number,
+    .token.constant,
+    .token.symbol {
+      color: #000;
+    }
+
+    .token.selector,
+    .token.attr-name,
+    .token.string,
+    .token.char,
+    .token.builtin,
+    .token.inserted {
+      color: #57a;
+    }
+
+    .token.operator,
+    .token.entity,
+    .token.url,
+    .token.variable {
+      color: #940;
+    }
+
+    .token.atrule,
+    .token.attr-value,
+    .token.keyword {
+      color: #060;
+    }
+
+    .token.regex,
+    .token.important {
+      color: #e90;
+    }
+
+    .token.variable,
+    .token.important,
+    .token.bold {
+      font-weight: bold;
+    }
+    .token.italic {
+      font-style: italic;
+    }
+
+    .token.deleted {
+      color: red;
+    }
+  }
+}
+
```

Comment:
Really great stuff. Probably missed this, just leaving the same comment as before:

> Good comments here, suggestion: because this is a large chunk of styles all pertaining to notebooks, I think it would be a good idea to create a separate file like `notebooks.scss` and import it into `styles.scss` similar to [featured-content.scss](https://github.com/dockstore/dockstore-ui2/blob/develop/src/featured-content.scss) which also styles external html (just for separation since these styles won't be used elsewhere):
> 
> https://github.com/dockstore/dockstore-ui2/blob/274e8deece1f8fdfbdc3f20b7ea5cb0f8a80aaea/src/styles.scss#L24
> 
> @import is actually going to be [phased out](https://sass-lang.com/documentation/at-rules/import) in a few years so it's recommended to use @use instead. Just tried it locally and styles rendered correctly. Eventually we should switch over our other @imports here as well

Reply:
Thanks for the gentle reminder, Kimberley, it had dropped off my radar.  I went with the `@import` syntax just because we use it elsewhere and know it to be problem free, we can convert them all to `@use` in a batch, someday...

Reply:
Np, just made a ticket to note this